pkg/controller/persistence/dgraph/main.go

```diff
@@ -0,0 +1,21 @@
+package dgraph
+
+import (
+	"fmt"
+)
+
+// Helper implementation for testing dgraph persistence.
+func main() {
+	fmt.Println("Hello World")
+	Open("127.0.0.1:9080")
+	err := CreateSchema()
+	if err != nil {
+		fmt.Println("Error while creating schema ", err)
+	}
+
+	uid, err := GetUId(Client, "default:Pod2", "isPod")
+	if err != nil {
+		fmt.Println("Error while fetching uid ", err)
+	}
+	fmt.Println("Uid is " + uid)
+}
```

Comment:
Instead of having this is as the main function, is it feasible to have this as a test?

Reply:
I will remove this file and write proper tests once major development for dgraph persistence is done.